tests/test_inscopiximagingextractor.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+import numpy as np
+import pytest
+from numpy import dtype
+from numpy.testing import assert_array_equal
+
+from roiextractors import InscopixImagingExtractor
+
+from .setup_paths import OPHYS_DATA_PATH
+
+
+@pytest.mark.skipif(sys.version_info <= (3, 8), reason="Requires Python 3.9 or higher")
+def test_inscopiximagingextractor_movie_128x128x100_part1():
+    file_path = OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_128x128x100_part1.isxd"
+    extractor = InscopixImagingExtractor(file_path=file_path)
+
+    assert extractor.get_num_frames() == 100
+    assert extractor.get_image_size() == (128, 128)
+    assert extractor.get_dtype() == dtype("float32")
+    assert extractor.get_sampling_frequency() == 10.0
+    assert extractor.get_channel_names() == ["channel_0"]
+    assert extractor.get_num_channels() == 1
+    assert extractor.get_video().shape == (100, 128, 128)
+
+
+@pytest.mark.skipif(sys.version_info <= (3, 8), reason="Requires Python 3.9 or higher")
+def test_inscopiximagingextractor_movie_longer_than_3_min():
+    file_path = OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_longer_than_3_min.isxd"
+    extractor = InscopixImagingExtractor(file_path=file_path)
+
+    assert extractor.get_num_frames() == 1248
+    assert extractor.get_image_size() == (33, 29)
+    assert extractor.get_dtype() == dtype("uint16")
+    np.testing.assert_almost_equal(extractor.get_sampling_frequency(), 5.5563890139076415)
+    assert extractor.get_channel_names() == ["channel_0"]
+    assert extractor.get_num_channels() == 1
+    assert extractor.get_video().shape == (1248, 33, 29)
+
+
+@pytest.mark.skipif(sys.version_info <= (3, 8), reason="Requires Python 3.9 or higher")
```

Comment:
I'm on the fence. I was keeping them on in case users run 3.8 tests. On the other hand, maybe it's best for them to fail to inform the user that inscopix is not supported on their system